### ISSUE TITLE
Empty configuration required

### ISSUE THREAD

**Issue by sondreb** [OR13/did-jwk#2]
The `create` method require an object (`{}`) and will crash if nothing is provided.

Within the constructor of DIDMethodClient, this is done:

`if (!config.documentLoader) {`

This needs to verify if config is null, which can happen, or is that not suppose to be allowed? If not allowed, an argument exception should be thrown instead of throw error: `Cannot read properties of undefined (reading 'documentLoader')`

**Comment by OR13** [comment]
Yeah, the devx is not great. I will be fixing this in the next version.
